components/cdc/src/observer.rs

```diff
@@ -411,7 +411,7 @@ mod tests {
 
         must_prewrite_put(&engine, k, b"v2", k, 2);
         must_get_eq(2, Some(b"v1".to_vec()));
-        must_rollback(&engine, k, 2);
+        tikv::storage::txn::commands::rollback::tests::must_success(&engine, k, 2);
```

Comment:
Can we organize and reexports these functions somewhere? Using like this way is ugly...

Reply:
> Can we organize and reexports these functions somewhere? Using like this way is ugly...

I'm still considering where should we put these functions in, maybe `storage::txn::commands::tests`.
And all `must_succeed` and `must_err` in the commands are very similar, I'm considering abstract them into one function, maybe this will make the reexports unnecessary, we'll see after some try.

Anyway, can we merge this pr now?